### PR TITLE
Amanda

### PR DESCRIPTION
**Pull request by AmandaNguye**
Add application opening dates
Add closed/open-ended FAQ question

### CODE REVIEW

**Comment by AmandaNguye** [comment]
After this is merged, i have edits to make for the FAQ format

File: src/pages/ApplyPage.tsx [hunk 1]
```diff
@@ -154,6 +154,18 @@ function ApplyPage() {
             <li>Additional team meetings may be arranged by your Project Manager when necessary</li>
             "
           />
+          <Accordion
+            title="How are projects directed?"
```
Comment (Jhappy77) [inline src/pages/ApplyPage.tsx:158]: Better title for question: "How do teams pick an idea for their project?"

File: src/pages/ApplyPage.tsx [hunk 2]
```diff
@@ -154,6 +154,18 @@ function ApplyPage() {
...
+            active={active}
+            setActive={setActive}
+            description="
+                The process of directing a projects either starts:
```
Comment (Jhappy77) [inline src/pages/ApplyPage.tsx]: Your Project Manager will pick one of the following ways to decide the team's project focus:

File: src/pages/ApplyPage.tsx [hunk 3]
```diff
@@ -154,6 +154,18 @@ function ApplyPage() {
...
+                </br>
+                <li>Focused: Project managers can choose to focus their project on a specific idea or concept from the get-go. These ideas are pitched at the first Tech Start meeting and members may then decide to join that team if they are interested! The teams then brainstorm adaptations and improvements to the idea, alongside the technologies and platforms to work with!</li>
```
Comment (Jhappy77) [inline src/pages/ApplyPage.tsx]: Maybe rephrase the answer to this question to be more tailored to general members, then have an answer more specific to PMs in a PM / Executive Team-specific FAQ section

**Review by Jhappy77 — CHANGES_REQUESTED** [review]

File: src/pages/ApplyPage.tsx (hunk 1 above)
Comment (AmandaNguye) [reply src/pages/ApplyPage.tsx:158]: Completed.

File: src/pages/ApplyPage.tsx (hunk 3 above)
Comment (AmandaNguye) [reply src/pages/ApplyPage.tsx]: Completed
